red/red_cog_info_json.md

```diff
@@ -23,18 +23,26 @@ A repository info.json sits at the root level of the repository (Before the cog
 ```
 
 ### Cog info.json
-A cog info.json sits in the cog folder (Besides the ``.py``). These are unique per cog. And can be changed per cog. The following format is used for them.
+A cog info.json sits in the cog folder (Besides the ``.py``). These are unique
+per cog. And can be changed per cog. The following format is used for them.
 
 ```json
 {
     "AUTHOR" : "Author name",
     "INSTALL_MSG" : "A install message that appears as soon as the end user does !cog install repo cogname",
     "NAME" : "The class name of the cog, MUST BE matching, including capitalization",
     "SHORT" : "A Short description, appears on !cog list repo",
-    "DESCRIPTION" : "A longer description, appears on !cog list repo cogname"
+    "DESCRIPTION" : "A longer description, appears on !cog list repo cogname",
+    "TAGS" : ["example", "tags", "list"],
+    "REQUIREMENTS" : ["tabulate", "Pillow"]
 }
 ```
 
+The `REQUIREMENTS` key is used when installing the cog. Before it is loaded,
+the installer ensures that those packages are installed via `pip`.
+
+The `TAGS` key is used on our official cogs library portal, [cogs.red](https://cogs.red/)
+
 #### Info.json maker:
 [Link](https://gist.github.com/Twentysix26/73a2a09238d6875b5cb7)
```

Comment:
looks good to me, but now the Info.json maker is out of date
shoud add a note or add the new sections to that as well before merge